### ISSUE TITLE
[Feature Request] Ability to pass external android controller like gamesir

### ISSUE THREAD

**Issue by gnisch** [breeze2/dsu-controller-guides#12]
Hello 
Is it possible to add feature to map buttons from external controller connected to phone like gamesir x2 pro or alternate?

**Comment by JesterofDoom13** [comment]
I'd also appreciate this feature.

**Comment by Ricardo199** [comment]
Dinput support would be greatly appreciated, even if it's probably a pipe dream, would make this app the best app out there, currently i have to switch between my physical gamepad and my phone, but having the ability to just have dinput would be amazing, or even just landscape support

**Comment by breeze2** [comment]
It seems similar to issue #9.
Do you want to turn the phone + external controller into a motion controller via DSUController?
I'll see what I can do.

**Comment by gnisch** [comment]
@breeze2 yes, exactly. In example, I have gamesir 2, which will be used as main controller, and phone will generate motion data. Ideally without using overlay.

**Comment by breeze2** [comment]
Please try version 2.3.0, which adds a feature: Local Gamepad.

**Comment by dutry** [comment]
@breeze2 

Is there a list how is every key in controllers mapped to the wiimotes? Or how can I make the setting?
